### PR TITLE
Avoid swapping directly to buffer

### PR DESCRIPTION
**Pull request by oldchili**

### CODE REVIEW

File: src/funnels/Swapper.sol [hunk 1]
```diff
@@ -100,13 +101,17 @@ contract Swapper {
             limits[src][dst].zzz = limit.zzz;
         }
 
-        uint256 prevDstBalance = GemLike(dst).balanceOf(buffer);
+        uint256 prevBalance = GemLike(dst).balanceOf(address(this));
```
Comment (sunbreak1211) [inline src/funnels/Swapper.sol]: Do we really care about the prev balance thinking we are now checking the swapper? Even if someone donated the dst token to the Swapper I wouldn't think it is a big deal.

Comment (oldchili) [reply src/funnels/Swapper.sol]: Removed it.

**Review by sunbreak1211 — APPROVED** [review]
Looks good!

**Comment by sunbreak1211** [comment]
Sorry I pushed to the wrong branch!

**Review by telome — APPROVED** [review]
Good to merge

**Pull request merged**